### PR TITLE
[KIECLOUD-453] - Adapt product artifacts definitions to CPaaS build

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

See: https://issues.redhat.com/browse/KIECLOUD-453

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[KIECLOUD-XYZ] Subject`, `[RHDM-XYZ] Subject` or `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Review by msmagnanijr — DISMISSED** [review]

File: tools/build-overrides/build-overrides.sh [hunk 1]
```diff
@@ -362,8 +358,7 @@ envs:
 - name: "CONTROLLER_DISTRIBUTION_ZIP"
   value: "${controller_distribution_zip}"
 artifacts:
-- name: "ADD_ONS_DISTRIBUTION_ZIP"
-  target: "add_ons_distribution.zip"
+- name: "rhpam_add_ons_distribution.zip"
```
Comment (jakubschwan) [inline tools/build-overrides/build-overrides.sh:361]:
```suggestion
- name: "rhdm_add_ons_distribution.zip"
```

File: tools/build-overrides/build-overrides.sh [hunk 2]
```diff
@@ -383,8 +378,7 @@ cat <<EOF > "${controller_overrides_json}"
   ],
   "artifacts": [
     {
-      "name": "ADD_ONS_DISTRIBUTION_ZIP",
-      "target": "add_ons_distribution.zip",
+      "name": "rhpam_add_ons_distribution.zip",
```
Comment (jakubschwan) [inline tools/build-overrides/build-overrides.sh:381]:
```suggestion
      "name": "rhdm_add_ons_distribution.zip",
```

File: tools/build-overrides/build-overrides.sh [hunk 3]
```diff
@@ -425,8 +418,7 @@ cat <<EOF > "${decisioncentral_overrides_json}"
 {
   "artifacts": [
     {
-      "name": "DECISION_CENTRAL_DISTRIBUTION_ZIP",
-      "target": "decision_central_distribution.zip",
+      "name": "rhpam_decision_central_distribution.zip",
```
Comment (jakubschwan) [inline tools/build-overrides/build-overrides.sh:421]:
```suggestion
      "name": "rhdm_decision_central_distribution.zip",
```

File: tools/build-overrides/build-overrides.sh [hunk 4]
```diff
@@ -458,8 +450,7 @@ EOF
                     log_info "Generating ${kieserver_overrides_yaml} ..."
 cat <<EOF > "${kieserver_overrides_yaml}"
 artifacts:
-- name: "KIE_SERVER_DISTRIBUTION_ZIP"
-  target: "kie_server_distribution.zip"
+- name: "rhpam_kie_server_distribution.zip"
```
Comment (jakubschwan) [inline tools/build-overrides/build-overrides.sh:453]:
```suggestion
- name: "rhdm_kie_server_distribution.zip"
```

File: tools/build-overrides/build-overrides.sh [hunk 5]
```diff
@@ -473,8 +464,7 @@ cat <<EOF > "${kieserver_overrides_json}"
 {
   "artifacts": [
     {
-      "name": "KIE_SERVER_DISTRIBUTION_ZIP",
-      "target": "kie_server_distribution.zip",
+      "name": "rhpam_kie_server_distribution.zip",
```
Comment (jakubschwan) [inline tools/build-overrides/build-overrides.sh:467]:
```suggestion
      "name": "rhdm_kie_server_distribution.zip",
```

**Review by jakubschwan — CHANGES_REQUESTED** [review]
Same changes needed as for #407

File: tools/build-overrides/build-overrides.sh (hunk 4 above)
Comment (spolti) [reply tools/build-overrides/build-overrides.sh:453]: thanks for this, too much ctrlC and V issues :D

**Review by Kaitou786 — APPROVED** [review]

**Review by msmagnanijr — APPROVED** [review]

**Review by jakubschwan — APPROVED** [review]

**Pull request merged**